app.js

```diff
@@ -5,6 +5,7 @@ const morgan = require('morgan');
 
 const dataSource = require('./models/appDataSource');
 const { globalErrorHandler } = require('./utils/error');
+const router = require('./routes/index');
```

Comment:
모듈 전체를 requrire하면 index.js를 기본으로 찾습니다.
```suggestion
const router = require('./routes');
```